react/store/createReturnRequest/components/ConfirmAndSubmit.tsx

```diff
@@ -28,8 +29,10 @@ export const ConfirmAndSubmit = ({ onPageChange, items }: Props) => {
     MutationCreateReturnRequestArgs
   >(CREATE_RETURN_REQUEST)
 
-  // temp state just to show request id on UI
-  const [requestId, setRequestId] = useState('')
+  const { navigate } = useRuntime()
+
+  const [showSubmittedStatus, setShowSubmittedStatus] = useState(false)
```

Comment:
Creating a state to control when a piece of component has to be visible is fine, but we can try to use some different approaches here that might be simpler and easier to maintain.

1. Instead of initiating `confirmationSatus` as `success` (which tells something that is not true for the application since the confirmation is not successfully yet) we can initiate it as an empty string. Then, we can control the visibility of the component using a check for falsy values to `confirmationStatus`.
2. `useMutation` returns a property called `called`. We can combine that with the loading key to control visibility to the component.